_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,112 @@
+
+You may have noticed we haven't done anything with A2 or A3 yet. If you followed the tutorial we will have set A2 Hardware Output to a Virtual Audio Device. If you select A2 on the Voicemeeter VAIO input, and the Microphone input, you will see down in the right corner the audio level bar for A2 moving. The reason we set up this Virtual Audio Devices is to pull into OBS.
+
+Picture of Audio bars moving after A1 and A2 set:
+
+
+### OBS Audio and alerts
+Lets get our audio into OBS! Open OBS and go to Settings → Audio. For this all we do is set Desktop Audio to Cable Input (VB-Audio Virtual Cable). Disable everything else since we are using Voicemeeter Banana to route and mix all our audio to one place.
+
+It should look a little something like this:
+
+![https://i.imgur.com/uKCbMLI.png](https://i.imgur.com/uKCbMLI.png)
+
+You might ask, what about alerts? I'm going to wager your alerts are being pulled into your OBS via browser source.  You set these up your normal way and when you do make sure to set Control Audio Via OBS. This will give you an Audio level bar in the Audio Mixer portion of OBS.
+
+![https://i.imgur.com/CrZdTZU.png](https://i.imgur.com/CrZdTZU.png)
+
+You will notice a cog options menu icon beside the specific Audio source in the Audio Mixer. Click this and you will see a menu and something that reads Advanced Audio Properties which opens a window like this:
+
+![https://i.imgur.com/cUmLUPo.png](https://i.imgur.com/cUmLUPo.png)
+
+Select Monitor and Output for the Alerts you want to hear.
+
+Then go into Settings→Audio→Advanced section→Monitoring Device: Set this to VoiceMeeter Aux Input (VB-Audio VoiceMeeter AUX VAIO):
+
+![https://i.imgur.com/VxwSMUG.png](https://i.imgur.com/VxwSMUG.png)
+
+Now go to your Voicemeeter Banana application and test some Alerts. You should see the Voicemeeter AUX audio bouncing around. Since we are pulling the audio into OBS via Browser Source and both Outputting and Monitoring. All we need is to push this audio to our headphones so we can hear when things happen. So select whichever A# is your headphones and test again to make sure you can hear it.
+
+I believe this should cover everything to get started with Voicemeeter Banana. However, if you have any trouble please join the [Library Discord](https://discord.streamsounds.tv/) to get the help you need with setting up Voicemeeter or if you have any questions about audio, audio mixing, microphones, OBS, streaming, and other things. We would love to have a chat about things and look forward to learning, teaching and assisting in many different things. Not just audio! Just like a library is full of random knowledge collected, we wish to grow our own knowledge and share it with others. So please come join us in the sharing.
```

Comment:
Use `[StreamSounds Discord Server]({{ site.streamsounds.discord }})` instead. This will make sure the URL still works in case we change it in the config file.